app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command_line: str) -> None:
+    _, source_name, destination_name = command_line.split(" ")
+
+    if source_name == destination_name:
+        return
+
+    with (open(source_name, "r") as source,
```

Comment:
as it told in README:
`Function take only one argument command, that is string with command cp`
You should to check this condition